packages/simple-ica/src/ibc_msg.rs

```diff
@@ -7,7 +9,14 @@ use serde::{Deserialize, Serialize};
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum PacketMsg {
-    Dispatch { msgs: Vec<CosmosMsg> },
+    Dispatch {
+        msgs: Vec<CosmosMsg>,
+        callback: Option<String>,
```

Comment:
This change introduces compile fail in a test. Can you check `cargo build --all-targets` in the repo root?

Reply:
Done in 7a22f0a. Hope you don't mind if I add commits here.